contracts/modules/common/RelayerModule.sol

```diff
@@ -105,7 +105,8 @@ contract RelayerModule is BaseModule {
         bytes32 signHash = getSignHash(address(this), address(_wallet), 0, _data, _nonce, _gasPrice, _gasLimit);
         require(checkAndUpdateUniqueness(_wallet, _nonce, signHash), "RM: Duplicate request");
         require(verifyData(address(_wallet), _data), "RM: Target of _data != _wallet");
-        uint256 requiredSignatures = getRequiredSignatures(_wallet, _data);
+        uint256 requiredSignatures;
+        (requiredSignatures,) = getRequiredSignatures(_wallet, _data);
```

Comment:
This can be written as `(uint256 requiredSignatures,) = getRequiredSignatures(_wallet, _data);` which I personally find more readable.